managed-services/webhosting/reference-content/cpanel-reference-content.mdx

```diff
@@ -46,7 +46,7 @@ If you need additional help setting up your Web Hosting plan, or have any questi
 
 * [Full Backups](https://docs.jetbackup.com/manual/cpanel/BackupTypes/fullBackups.html)
 * [File Backups](https://docs.jetbackup.com/manual/cpanel/BackupTypes/fileBackups.html)
-* [Cron Job Backups](https://docs.jetbackup.com/manual/cpanel/BackupTypes/cronJobBackups.html)
+* [CronJob Backups](https://docs.jetbackup.com/manual/cpanel/BackupTypes/cronJobBackups.html)
```

Comment:
To discuss, but I think "CronJob" is a K8s way to write it. In reference to Linux, it should be "cron job" afaik

Reply:
```suggestion
* [Cron Job Backups](https://docs.jetbackup.com/manual/cpanel/BackupTypes/cronJobBackups.html)
```